pkg/TIPA.py

```
@@ -158,593 +185,527 @@ class MessageFunc():
     '''
-    def __init__(self, img, mouse_pos, display_info_init, gui_queue):
+
+    # Constants for UI coordinates
+    INVENTORY_COORDS = {
+        'eyewear_text': {
+            'x1': 598,
+            'y1': 421,
+            'x2': 692,
+            'y2': 441
+        }
+    }
```

Ruff (0.8.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)